src/main/java/adapters/login_leaderboard/RegisterUserController.java

```diff
@@ -1,8 +1,33 @@
 package adapters.login_leaderboard;
 
+import use_cases.login_leaderboard.IRegisterUserInputBoundary;
+
 /**
  * Transforms the information from RegisterScreen to be accessible for RegisterUser.
  * Passes username, password, and email address.
  */
 public class RegisterUserController {
+
+    /***
+     * adds in the use case to be processed as an attribute.
+     */
+    private final IRegisterUserInputBoundary useCaseInteractor;
+
+    /***
+     * Constructor for the class. Self explanatory.
+     * @param useCaseInteractor the current use case to be performed.
+     */
+    public RegisterUserController(IRegisterUserInputBoundary useCaseInteractor) {
+        this.useCaseInteractor = useCaseInteractor;
+    }
+
+    /***\
```

Comment:
is there a reason for the \ at the end and the three * starting most of your javadocs when two is enough?